ksql-rest-app/src/test/java/io/confluent/ksql/rest/server/computation/CommandStoreTest.java

```diff
@@ -292,14 +290,121 @@ public void shouldDistributeCommand() throws ExecutionException, InterruptedExce
     assertThat(record.value().getOriginalProperties(), equalTo(ksqlConfig.getAllConfigPropsWithSecretsObfuscated()));
   }
 
+  @Test
+  public void shouldIncludeTopicOffsetInSuccessfulQueuedCommandStatus()
+      throws InterruptedException, ExecutionException {
+    // Given:
+    final CommandStore commandStore = createCommandStoreThatAssignsSameId(commandId);
+
+    expect(commandProducer.send(anyObject(ProducerRecord.class))).andReturn(future);
```

Comment:
Looks like this wiring could be done in `setUp` or `createCommandStoreThatAssigns...`

Reply:
I'll take a look at the new tests.